pkg/datapath/linux/node.go

```diff
@@ -429,12 +429,18 @@ func (n *linuxNodeHandler) enableIPsec(newNode *node.Node) {
 		new4Net := &net.IPNet{IP: newNode.IPv4AllocCIDR.IP, Mask: newNode.IPv4AllocCIDR.Mask}
 		if newNode.IsLocal() {
 			n.replaceNodeIPSecInRoute(new4Net)
+			ciliumInternalIPv4 := newNode.GetCiliumInternalIP(false)
+			if ciliumInternalIPv4 != nil {
+				ipsecLocal := &net.IPNet{IP: n.nodeAddressing.IPv4().Router(), Mask: n.nodeAddressing.IPv4().AllocationCIDR().Mask}
+				ipsecIPv4Wildcard := &net.IPNet{IP: net.ParseIP("0.0.0.0"), Mask: net.IPv4Mask(0, 0, 0, 0)}
```

Comment:
small nit, consider making this a "static" / "constant" variable in this file

Reply:
will do.